```
(venv) [root@5ebc31bfda5b /]# pip install omero-py
Collecting omero-py
  Using cached https://files.pythonhosted.org/packages/78/0b/2888ea12547b17058486e5b8c1e07ac4e3ec44c6cb63009bfc8f592230c3/omero-py-5.5.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-v8r40vzp/omero-py/setup.py", line 16, in <module>
        from StringIO import StringIO
    ModuleNotFoundError: No module named 'StringIO'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-v8r40vzp/omero-py/
You are using pip version 9.0.3, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
(venv) [root@5ebc31bfda5b /]# 
```

This does not work as written. In fact, I am blocked now, ran out of ideas.

![Screen Shot 2020-01-08 at 10 41 14](https://user-images.githubusercontent.com/2478303/71971872-6fc9ac80-3203-11ea-8cad-aa7f0b53d570.png)

--- a/common/conf.py
+++ b/common/conf.py
@@ -210,6 +210,7 @@ rst_epilog = """
 .. _Libjpeg: http://libjpeg.sourceforge.net/
 .. _Django: https://www.djangoproject.com/
 .. _PyPI: https://pypi.org
+.. _Conda: https://docs.conda.io/en/latest/
 
 .. |SSH| replace:: :abbr:`SSH (Secure Shell)`
 .. |VM| replace:: :abbr:`VM (Virtual Machine)`

--- a/omero/users/cli/installation.rst
+++ b/omero/users/cli/installation.rst
@@ -10,13 +10,13 @@ We assume that you have already installed Python 3.6.
 We recommend installing omero-py and the |CLI| plugins
 in a Python virtual environment.
 You can either create one using ``venv`` or ``conda`` (preferred).
-If you opt for `conda <https://docs.conda.io/en/latest/>`_, you will need
+If you opt for Conda_, you will need
 to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
 
 To create a virtual env and install ``omero-py`` using venv::
 
-    python3.6 -mvenv venv
-    . venv/bin/activate
+    python3.6 -m venv py3_venv
+    . py3_venv/bin/activate
     pip install omero-py
 
 To create a virtual env and install ``omero-py`` using conda (preferred)::

--- a/omero/users/cli/installation.rst
+++ b/omero/users/cli/installation.rst
@@ -4,27 +4,48 @@ Installation
 .. note:: The |CLI| is currently untested on Windows
     but may be supported in the future.
 
-We recommend installing in a Python virtual environment.
-Since OMERO 5.6, only Python 3 is supported::
+Since OMERO 5.6, only Python 3 is supported.
+We assume that you have already installed Python 3.6.
+
+We recommend installing omero-py and the |CLI| plugins
+in a Python virtual environment.
+You can either create one using ``venv`` or ``conda`` (preferred).
+If you opt for `conda <https://docs.conda.io/en/latest/>`_, you will need
+to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
+
+To create a virtual env and install ``omero-py`` using venv::
 
     python3.6 -mvenv venv
     . venv/bin/activate
     pip install omero-py
 
-    # omero command is now available
+To create a virtual env and install ``omero-py`` using conda (preferred)::
+
+    conda create -n myenv python=3.6
+    conda activate myenv
+    conda install -c ome zeroc-ice36-python 
+    conda install -c ome omero-py 
+
+The ``omero`` command is now available::
+
     omero login
 
-The |CLI| installed in this way provides all functionalities except import.
-Import requires JARs in the OMERO.server package to be available under
+The |CLI| installed in this way provides all functionalities except the ``import`` functionality.
+
+The ``import`` functionality requires to have a supported version of Java e.g. Java, 11 Ice |iceversion| installed
+and also requires JARs in the OMERO.server package to be available under
 the :envvar:`OMERODIR` directory.
 
-You need to download the OMERO.server zip from under artifacts at
-:downloads:`downloads page <>`. Unzip the zip file and set ``$OMERODIR``
-to the unzipped directory::
+To install Java and Ice, go to :doc:`/sysadmins/unix/server-installation` and select the walk-through corresponding to your OS.
+
+To make the JARs available, you need to:
+
+* download the OMERO.server zip from under artifacts at :downloads:`downloads page <>`
+* unzip the zip file 
+* set ``$OMERODIR`` to the unzipped directory::
 
     export OMERODIR=/path/to/OMERO.server-x.x.x-ice36-bxx
 
-    # now you can import
-    omero import /path/to/image.tiff
+* the ``import`` functionality is now available::
 
-For more details, see :doc:`/sysadmins/unix/server-installation`.
+    omero import /path/to/image.tiff

